moveit_cartesian_plan_plugin/CMakeLists.txt

```diff
@@ -11,21 +13,54 @@ find_package(catkin REQUIRED COMPONENTS
   rqt_gui_cpp
   rviz
   visualization_msgs
-  moveit_ros_move_group
-  moveit_ros_planning_interface
   tf
   tf_conversions
+  moveit_ros_move_group
+  moveit_ros_planning_interface
+  moveit_ros_warehouse
+  moveit_ros_perception
 )
+#  moveit_ros_move_group
+#  moveit_ros_planning_interface
+# find_package(Qt4 COMPONENTS QtCore QtGui REQUIRED)
 
-find_package(Qt4 COMPONENTS QtCore QtGui REQUIRED)
+find_package(Eigen3 REQUIRED)
 
-include(${QT_USE_FILE})
+# Eigen 3.2 (Wily) only provides EIGEN3_INCLUDE_DIR, not EIGEN3_INCLUDE_DIRS
+if(NOT EIGEN3_INCLUDE_DIRS)
+  set(EIGEN3_INCLUDE_DIRS ${EIGEN3_INCLUDE_DIR})
+endif()
 
-## I prefer the Qt signals and slots to avoid defining "emit", "slots",
-## etc because they can conflict with boost signals, so define QT_NO_KEYWORDS here.
+# Qt Stuff
+if(rviz_QT_VERSION VERSION_LESS "5")
+	find_package(Qt4 ${rviz_QT_VERSION} REQUIRED QtWidgets QtCore QtGui)
+	include(${QT_USE_FILE})
+  set(QT_LIBRARIES ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY})
+else()
+  find_package(Qt5Widgets REQUIRED)
+  find_package(Qt5Concurrent REQUIRED)
+	set(QT_LIBRARIES Qt5::Widgets Qt5::Concurrent)
+endif()
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOMOC ON)
 add_definitions(-DQT_NO_KEYWORDS)
+add_definitions(-std=c++11) 
```

Comment:
This is redundant as there already is a -std=c++11 compile option at the top of the file